project_ideas/app/managers.py

```diff
@@ -0,0 +1,33 @@
+from django.contrib.auth.base_user import BaseUserManager
+from django.utils.translation import ugettext_lazy as _
+
+
+class CustomUserManager(BaseUserManager):
+    
+    def create_user(self, email, password, **extra_fields):
+        if not email:
+            raise ValueError(_('The Email must be set'))
+        email = self.normalize_email(email)
+        user = self.model(email=email, **extra_fields)
+        user.set_password(password)
+        user.save()
+        return user
+
+    def _create_user(self, email, **extra_fields):
+        if not email:
+            raise ValueError(_('The Email must be set'))
+        email = self.normalize_email(email)
+        user = self.model(email=email, **extra_fields)
+        user.save()
+        return user
+
```

Comment:
Add comment to the function or add docstring to tell the usage of the function. I cannot tell the difference between create_user and _create_user function if i am not aware of it.

Reply:
Actually forgot to remove the other one (_create_user). Completed this